eng/helix/helix.proj

```diff
@@ -12,7 +12,8 @@
   <Import Project="..\Build.props" />
 
   <ItemGroup>
-    <HelixTargetQueue Include="@(HelixAvailableTargetQueue)" />
+    <HelixTargetQueue Condition="'$(UseHelixOpenQueues)' == 'true'" Include="@(HelixAvailableTargetQueue)" />
```

Comment:
OK this cements my previous assertion above;  while this today is almost certainly valid, there definitely do exist queues which are only internal and only open;  having explicit lists (and perhaps having one be shorter or longer depending on required turn around time) is a better idea.  If you don't mind the brittleness this will probably work

Reply:
Well we also have the queue ids explicitly in our attributes in test sources for what helix queues to skip, so this is a larger change that will need a dedicated PR and needs to be done carefully.  I don't think we should try to tackle it as part of adding the api token